apps/bfd-server/dev/bfd-data-dictionary/app/dd-transformer/runner.ps1

```diff
@@ -0,0 +1,62 @@
+﻿#################################################################
+# To use...
+# 
+# 1. modify dataDictRoot and outputRoot variable expressions below to point to approp locations on local env
+# 2. run all root var expressions (highlight expressions + F8)
+# 3. run desired trasnformer expression (e.g. CSV, JSON, HTML)
+#################################################################
+
+
+########################
+# setup root variables
+########################
+
+$dataDictRoot = "c:\Users\621222\dev\projects\beneficiary-fhir-data\apps\bfd-server\dev\bfd-data-dictionary"
+$outputRoot = "c:\output"
```

Comment:
Likely better off just leaving these blank (or at best putting an example) here, since these are A. windows paths and B. that root is specific to your directory

Ideally a second pass at these scripts will have the root and output be either inputs or just a relative path in the folder structure.

Reply:
I have removed the runner.ps Powershell script altogether and placed the relevant content under "Running Locally" as suggested below.